inc/commondbtm.class.php

```diff
@@ -226,6 +226,8 @@ function post_getFromDB() {
     * Actions done to not show some fields when geting a single item from API calls
     *
     * @param array $fields Fields to unset undiscloseds
```

Comment:
Yes, of course !
But couldn't documented it (and maybe update CS too) ?

I think it's an important information for a dev.

Your opinion ?